frontend/src/bundles/chats/components/chat-sidebar/chat-sidebar.tsx

```diff
@@ -26,6 +34,29 @@ const ChatSidebar = ({
     className,
     onLoadCurrentChat,
 }: Properties): JSX.Element => {
+    const [usersChats, setUsersChats] = useState<
+        ChatPreviewResponseDto[] | null
+    >(null);
+    const [search, setSearch] = useState<string | null>(null);
```

Comment:
also, why do we need this local state?

Reply:
We can't change the state of chats in store, as we wont be able to find something right after first search, since the state is changed. I can create one additional state in store but i dont think this is necessary. Maybe there is a better way to do it?